saleor/graphql/order/bulk_mutations/orders.py

```diff
@@ -0,0 +1,47 @@
+import graphene
+
+from ....order import OrderEvents, models
+from ....order.utils import cancel_order
+from ...core.mutations import BaseBulkMutation
+from ..mutations.orders import clean_order_cancel
+
+
+class OrderBulkCancel(BaseBulkMutation):
+    class Arguments:
+        ids = graphene.List(
+            graphene.ID,
+            required=True,
+            description='List of orders IDs to cancel.')
+        restock = graphene.Boolean(
+            required=True,
+            description='Determine if lines will be restocked or not.')
+
+    class Meta:
+        description = 'Cancels orders.'
+        model = models.Order
+
+    @classmethod
+    def clean_instance(cls, info, instance):
+        clean_order_cancel(instance)
+
+    @classmethod
+    def user_is_allowed(cls, user, input):
```

Comment:
We've just got rid of `input` argument name; in bulk mutations we use `_ids` for this argument.